### DISCUSSION TITLE
Using Unicode Character “🠝” (U+1F81D) in the ggplot xaxis title

### DISCUSSION THREAD

**Discussion by zhengchencai** [quarto-dev/quarto-cli#9591] [Q&A] [answered]
### Description

Hello,

I am trying to use Unicode Character “🠝” (U+1F81D) in one of the figure's x axis title. When I run the R chunk in qmd, it returns me the correct character.  The ggplot just uses lab(x = "Pr(outcome) 🠝"). 

![image](https://github.com/quarto-dev/quarto-cli/assets/29985544/a433f06d-7138-43dc-ac9d-ec67609a4151)

However, when I render the whole HTML, the 🠝 is not recognized.  Is there something I should set in YAML?

![image](https://github.com/quarto-dev/quarto-cli/assets/29985544/2d99db42-8383-4a63-9ec5-42681e0c0705)

I also tried the HTML entity shown here https://www.compart.com/en/unicode/U+1F81D, but it is still not rendering correctly. 

Thank you very much.

**Comment by mcanouil** [comment]
You need to use a graphic device and have the font that can use that unicode.
That's not really related to Quarto.

For instance, I can't even see that unicode on MacOS but I can see it on Windows.

````qmd
---
title: "Quarto Playground"
format: html
knitr:
  opts_chunk: 
    dev: ragg_png
---

This is a playground for Quarto.

```{r}
library(ggplot2)

ggplot() + geom_blank() + labs(title = "Hello, Quarto \u03a9")
```
````

**Reply by zhengchencai** [reply]
Hi @mcanouil , thanks for your suggestion. I do use ragg to save figures after ggplot. \u03a9 works but “🠝” (U+1F81D) does not. Probably I am not use the right code. I will double-check. Thanks again.

**Reply by mcanouil** [reply, marked as answer]
It's a matter of font being used.
You need to use a font that has those unicode you want.

Unfortunately, this is unrelated to Quarto, thus out of Quarto support scope.

**Reply by zhengchencai** [reply]
No worries, thanks a lot. You have already explained to me the causal and I can search to fix it. Thanks again.

**Reply by zhengchencai** [reply]
Hi @mcanouil, I tied this and it works now. "Pr(outcome) \U0001F81D".
